aws/resource_aws_apigatewayv2_api.go

```diff
@@ -278,6 +322,13 @@ func resourceAwsApiGatewayV2ApiUpdate(d *schema.ResourceData, meta interface{})
 		}
 	}
 
+	if d.HasChange("body") {
+		err := resourceAwsAPIGatewayV2ImportOpenAPI(d, meta)
+		if err != nil {
+			return fmt.Errorf("error update API Gateway v2 API (%s) with OpenAPI spec: %s", d.Id(), err)
```

Comment:
```suggestion
			return err
```

The relevant error information was captured in `resourceAwsAPIGatewayV2ImportOpenAPI` so there's no need to wrap here.